_posts/2019-07-10-cockpit-198.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Cockpit 198
+author: pitti
+date: 2019-07-10
+tags: cockpit linux
+slug: cockpit-198
+category: release
+summary: Cockpit with PatternFly 4 user interface design update
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 198.
+
+### New user interface design
+
+Cockpit moved to the [PatternFly 4](https://pf4.patternfly.org/components/Button/examples/)
+User Interface design, including the
+[Red Hat Text and Display fonts](https://github.com/RedHatOfficial/RedHatFont).
+This aligns Cockpit with other web user interfaces that use [PatternFly](https://www.patternfly.org/),
+such as [OpenShift 4](https://www.redhat.com/en/openshift-4).
+
+![login page](/images/login-pf4.png)
+
+![system page](/images/system-pf4.png)
+
+### SELinux: Show changes
+
+The SELinux page now has a new section "System Modifications" which shows all
+policy settings that were made to the system (with Cockpit or otherwise):
+
+![SELinux modifications](/images/selinux-modifications.png)
+
+The "View Automation Script" link will show a dialog with a Shell script that
+can be used to apply the same changes to other machines:
+
+![SELinux automation script](/images/selinux-autoscript.png)
+
+### Machines: Deletion of Virtual Networks
+
+On the Machines page's "Virtual Networks" page, networks can now be deleted:
```

Comment:
> Machines page's "Virtual Networks" page

oy

Maybe something like "The Virtual Networks section of the Machines page now supports deleting networks"?